{{project_name}}/.github/workflows/docs.yml.jinja

```diff
@@ -8,15 +8,11 @@ permissions:
 
 env:
   DBT_PROFILES_DIR: transform/ci
-{% if dbt_target == 'Snowflake' %}
-{% raw %}
```

Comment:
Here and elsewhere: the `{% raw %}` blocks are still necessary. This is because the copier template uses jinja, and we need to tell it that these lines should *not* be templated. So, for instance, the template shouldn't be looking for a variable names `SNOWFLAKE_ACCOUNT`

Reply:
Got it! Ty